ivadomed/inference.py

```diff
@@ -16,6 +16,31 @@
 from ivadomed import training as imed_training
 
 
+def get_preds(context, model, fname_model, onnx_inference, model_params, cuda_available, batch):
```

Comment:
I would recommend adding comments for the refactored/extracted function here, in addition to start introduce type hints. @gisellemartel I can take over the PR and the branch if you don't have to wrap this up. 

While the numerous parameter is a bit concern, it does at least alleviate the original cognitive burdens compare to before so I think this is a step in the right direction and should be further continued to simplify the inference process. 

I can help continue this effort in future PRs.

Reply:
Thank you @dyt811! I can take care of adding some comments to the functions.